### PR TITLE
fix hover style for char count field

### PR DESCRIPTION
**Pull request by ByronEricPerez**
## Fixes

Issue Number: https://github.com/lbryio/lbry-desktop/issues/7564

## What is the current behavior?
image

## What is the new behavior?
image

## PR Checklist

<details><summary>Toggle...</summary>

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I added a line describing my change to CHANGELOG.md
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

</details>

### CODE REVIEW

**Comment by ByronEricPerez** [comment]
Hi @jessopb  all good with this, do you need me to change something?

**Comment by jessopb** [comment]
I think this would be cleaner if it weren't class "button--file-action". It's not completely clear what file-action is, but this is a comment action or something. I think it deserves its own class. button--comment-icons, perhaps.

**Comment by ByronEricPerez** [comment]
Hello, @jessopb  I have already sent the changes, is this ok?

**Comment by jessopb** [comment]
@ByronEricPerez I think it just needs a clean class that takes the place of button--file-actions because it's not a "file-action".

**Comment by ByronEricPerez** [comment]
Hello @jessopb , it is not entirely clear to me, would it be to rename the class?

File: ui/component/common/form-components/form-field.jsx [hunk 1]
```diff
@@ -261,7 +261,7 @@ export class FormField extends React.PureComponent<Props> {
                 {!noEmojis && openEmoteMenu && (
                   <Button
                     type="alt"
-                    className="button--file-action"
+                    className="button--file-action button--comment-icons"
```
Comment (jessopb) [inline ui/component/common/form-components/form-field.jsx:264]: this should not need button--file-action
One class should be sufficient to specify what this should look like.
button--comment-icons should contain all that it needs.

File: ui/scss/component/_button.scss [hunk 2]
```diff
@@ -378,13 +383,52 @@ a.button--alt {
   }
 
   &:hover {
+    .button__label {
+      color: var(--color-link);
+    }
+  }
+}
+
+.button--comment-icons {
```
Comment (jessopb) [inline ui/scss/component/_button.scss:392]: I know it's not super D.R.Y. but I would duplicate the button--file-action class contents here.

Then we can get rid of commented out lines.

**Review by jessopb — CHANGES_REQUESTED** [review]

**Comment by ByronEricPerez** [comment]
Hi @jessopb , would that be ok?

File: ui/scss/component/_button.scss [hunk 3]
```diff
@@ -378,12 +378,52 @@ a.button--alt {
...
+  @extend .button--alt;
+  color: var(--color-text);
+  background-color: transparent;
+  margin-right: var(--spacing-m);
+  padding: 0 var(--spacing-xxs);
+  height: initial;
+  padding: 5px;
```
Comment (jessopb) [inline ui/scss/component/_button.scss:394]: There's a duplicate padding: line here. Also, whenever possible, we're trying to use vars for spacing, as above. Does one of those work?

**Review by jessopb — COMMENTED** [review]
Yeah, this is better. Just a couple more comments.

**Review by jessopb — APPROVED** [review]
This looks good!

**Pull request merged**